book-store/src/Components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC, useState } from 'react';
+import { IconButton, Icon } from '@mui/material';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import SearchIcon from '@mui/icons-material/Search';
+import Logo from '../../Images/booktown-logo.png';
+import './NavBar.css';
```

Comment:
please consider making it a module, so the styles you've set will not affect other components